code/modules/fish/fish_items.dm

```diff
@@ -199,6 +199,15 @@ var/global/list/fish_items_list = list("goldfish" = /obj/item/weapon/fish/goldfi
 	hitsound = 'sound/weapons/bite.ogg'
 	force = 3
 
+/obj/item/weapon/fish/babycarp/attackby(var/obj/item/O, var/mob/user as mob)
+	if(istype(O, /obj/item/weapon/kitchenknife) || istype(O, /obj/item/weapon/butch) || istype(O, /obj/item/weapon/scalpel) || istype(O, /obj/item/weapon/kitchen/utensil/knife))
```

Comment:
we do, but it's going to allow for weird things like gutting fish with a mining drill or wirecutters.